Resources/Prototypes/tags.yml

```diff
@@ -308,6 +308,9 @@
 - type: Tag
   id: MicrowaveSelfUnsafe
 
+- type: Tag
+  id: Monkey
+
```

Comment:
do monkeys actually have this tag

Reply:
Nope! I should have probably added that.